contracts/common.sol

```diff
@@ -18,4 +19,20 @@ library Common {
     ) internal pure returns(uint24) {
         return uint24(weaponMultiplier.mulu(basePower).add(bonusPower));
     }
-}
\ No newline at end of file
+
+    function getPowerAtLevel(uint8 level) internal pure returns (uint24) {
+        return Common.getPowerAtLevel(level);
```

Comment:
recursive call. I think you copied the code from characters

Reply:
Yah, typo.  Will remove the cut/paste snafu.